src/components/Panel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+export default class Panel extends React.Component {
```

Comment:
Likewise, this is more properly constructed as a stateless functional component.